front/src/common/Map/WarpedMap/getWarping.ts

```diff
@@ -0,0 +1,58 @@
+import bbox from '@turf/bbox';
+import simplify from '@turf/simplify';
+import { lineString } from '@turf/helpers';
+import { BBox2d } from '@turf/helpers/dist/js/lib/geojson';
+import { Feature, FeatureCollection, Geometry, LineString, Position } from 'geojson';
+
+import { extendLine, getSamples } from 'common/Map/WarpedMap/core/helpers';
+import { getQuadTree } from 'common/Map/WarpedMap/core/quadtree';
+import {
+  featureToPointsGrid,
+  getGridIndex,
+  getGrids,
+  pointsGridToZone,
+  straightenGrid,
+} from 'common/Map/WarpedMap/core/grids';
+import { clipAndProjectGeoJSON, projectBetweenGrids } from 'common/Map/WarpedMap/core/projection';
+
+export type WarpingFunction = ReturnType<typeof getWarping>['transform'];
+
+export default function getWarping(warpPath: Feature<LineString>) {
+  // Simplify the input path to get something "straighter", so that we can see
```

Comment:
I'm not sure about this comment
```suggestion
  // Simplify the input path to get something "straighter", so that we can not see
```

Reply:
I can try to reformulate, but my version is good:
- If we warp along the exact given path, on the "straightened map", the input path will appear as a straight line.
- We simplify the path to warp along so that in the final "straightened map", the input path appears as still a little curvy (and it's intended)

So yes, we simplify the input path so that we can still see small curves of the initial path in the end.

Reply:
Ok got it, thanks !